docs/manual/article-management/content-elements/file-elements.en.md

```diff
@@ -25,34 +34,95 @@ The special feature of Contao is that this download link also works with protect
 
 **Link title:** The link title is inserted as an `title` attribute in the HTML markup.
 
+
+{{< version "4.13" >}}
+
+### Preview settings
+
+{{% notice info %}}
+The PHP extension Imagick must be installed on the server to enjoy this feature.
```

Comment:
```suggestion
The PHP extension Imagick or Gmagick must be installed on the server to enjoy this feature.
```

Reply:
Doesn't ImageMagick itself also have some minimum requirements?

Reply:
Which formats are supported depends on the configuration of imagemagick, but it is probably best to solve this by showing the supported formats as suggested in https://github.com/contao/contao/issues/4122